src/text.ts

```diff
@@ -98,8 +123,17 @@ export const requestPhoneCode = async ({
       return false
     }
 
-    await PhoneCode.create({ phone, code })
-    await sendText({ body, to: phone, logger })
+    await PhoneCode.create({ phone, code, sms_provider })
+
+    
+    if (sms_provider === "twilio") {
+      await sendTwilioText({ body, to: phone, logger })
+    } else if (sms_provider === "smsala") {
+      await sendSMSalaText({ body, to: phone, logger })
```

Comment:
this could be refactored: `{ body, to: phone, logger }` and declared before the branch to avoid the repetition